cmd/autoupdate/git.go

```diff
@@ -94,7 +94,7 @@ func updateGit(ctx context.Context, pckg *packages.Package) ([]newVersionToCommi
 		}
 	}
 
-	return newVersionsToCommit, latestGitVersion
+	return newVersionsToCommit, version(lastExistingVersion)
```

Comment:
So `package.json` is first updated on the next run?

Reply:
it should all be fixed with one run (if my thinking is correct)!

Reply:
`lastExistingVersion := git.GetMostRecentExistingVersion(...)` is called before `newVersionsToCommit = doUpdateGit(...)`, so I don't think that is correct?

Reply:
Oh, I think you are correct :)